### ISSUE TITLE
When using dump, results are duplicated

### ISSUE THREAD

**Issue by luanfreitasdev** [tweakphp/tweakphp#41]
**Describe the bug**
Duplicate results when using dump

```php
dump([
    'name' => 'Luan',
    'created_at' => now()
])
```

**To Reproduce**

**Expected behavior**

**Screenshots**

Without dump: v
![image](https://github.com/user-attachments/assets/76456372-88a0-447e-bec5-c61d541f35e0)

With dump:
![image](https://github.com/user-attachments/assets/8ebdf769-9c1e-4c5a-b42b-537b71ac9daa)

**Desktop (please complete the following information):**
 - OS: [e.g. MacOS 10.2]
 - App Version [e.g. 0.2.0]

**Additional context**

**Comment by AleWasser** [comment]
Hey guys!
I have been trying to replicate the issue, but I am not sure if this is a bug.
Isn't that the way how the REPL output works, returning the value from the latest expression?

If we want to avoid this then we can return a null value to our input, like this:
```
//* src/main/client.ts
export const localExec = async (event: Electron.IpcMainEvent, data: { code: string; php: string; path: string }) => {
  //...
  let input = data.code.replaceAll('<?php', '')
  input += '; return null;'
  const code = btoa(input)
  //...
}
```
<img width="1582" alt="Screenshot 2024-12-29 at 2 26 47 AM" src="https://github.com/user-attachments/assets/e6e4e7bd-e7c1-4a38-abe2-d9234c581951" />

Or is it there any flag we can add here to avoid this?
```
const command = `${phpPath} ${getLocalPharClient()} ${path} execute ${code}  --do-not-show-me-extra-code`
```

**Comment by saeedvaziry** [comment]
@AleWasser if we return null (which btw it has to be in the tweakphp-client repo) then you will lose the last expression result.

for example if you execute:

```
collect([]);
```

or

```
$a = 1;
```

it won't print anything

**Comment by AleWasser** [comment]
@saeedvaziry Yeah.. I see. What should we do to handle this? Should we manipulate the output in the `execute` method?
Looks like this will only happen if there are just dumps in the editor 🤔
I mean, the duplication won't happen if we have this code:
```PHP
dump([
    'name' => 'Ale',
    'created_at' => now()
]);

$a = 1;
```
As it will return "`> 1`"

**Comment by saeedvaziry** [comment]
Ideally if there is an output in the execution then there is no need to print the last expression's result

**Comment by saeedvaziry** [comment]
Closing this as fixed. it will be included in the next update

![Image](https://github.com/user-attachments/assets/516a8620-93d8-4daf-beb3-98b9159b908b)
